src/mapd-crossfilter.js

```diff
@@ -1156,11 +1135,23 @@ export function replaceRelative(sqlStr) {
             subExpression += typedValue + " = ANY " + dimArray[e]
           } else if (Array.isArray(typedValue)) {
             if (typedValue[0] instanceof Date) {
-              const min = formatDateRangeLowerBound(typedValue[0])
-              const max = formatDateRangeUpperBound(typedValue[1])
-              const dimension = dimArray[e]
-              subExpression +=
-                dimension + " >= " + min + " AND " + dimension + " <= " + max
+              const dateRangeUpperBound = moment(typedValue[1])
+
+              if (dateRange.isSame(typedValue[0])) {
```

Comment:
I can't find where `dateRange` is defined?

Reply:
Whoa, good catch. I'll fix this. Also means I must not have been hitting this in a test case - I'll fix that too.